### PR TITLE
Add doc of screen recording

### PR DESCRIPTION
**Pull request by KazuCocoa**
## Proposed changes

I noticed screen recording feature doesn't in command list.
_android video recording #11024_

Almost description comes from Java/Python/Ruby/JS client's description.

## Types of changes

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: commands-yml/commands/device/recording-screen/start-recording-screen.yml [hunk 1]
```diff
@@ -0,0 +1,118 @@
+---
+name: Start Record Screen
+short_description: Returns the information of the system state which is supported to read as like cpu, memory, network traffic, and battery
```
Comment (imurchie) [inline commands-yml/commands/device/recording-screen/start-recording-screen.yml]: This doesn't seem right.

File: commands-yml/commands/device/recording-screen/start-recording-screen.yml [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+    |
+      // TODO PHP sample
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "https://static.javadoc.io/io.appium/java-client/6.1.0/io/appium/java_client/screenrecording/CanRecordScreen.html#startRecordingScreen-T-"
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L3412"
+  ruby: "https://www.rubydoc.info/github/appium/ruby_lib_core/Appium/Core/Android/Device#start_recording_screen-instance_method"
```
Comment (imurchie) [inline commands-yml/commands/device/recording-screen/start-recording-screen.yml]: Why is this in Android? Doesn't it work for iOS too?

File: commands-yml/commands/device/recording-screen/stop-recording-screen.yml [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+                   fit into the available process memory.
+    - name: username
+      description: The name of the user for the remote authentication.
+    - name: password
+      description: The password for the remote authentication.
+    - name: method
+      description: The http multipart upload method name. The 'PUT' one is used by default.
+  response:
+    - type:
+      description: The available performance data types (cpuinfo|batteryinfo|networkinfo|memoryinfo)
```
Comment (imurchie) [inline commands-yml/commands/device/recording-screen/stop-recording-screen.yml]: ?

Comment (KazuCocoa) [reply commands-yml/commands/device/recording-screen/stop-recording-screen.yml]: Oh... Sorry for the remaining copy&paste...

File: commands-yml/commands/device/recording-screen/start-recording-screen.yml (hunk 2 above)
Comment (KazuCocoa) [reply commands-yml/commands/device/recording-screen/start-recording-screen.yml]: Yes, it works both platform. Ruby has 2 URLs for each platform. But generating documentation command raised an error then. I'll take a look at the script and update this line/or the script.

File: commands-yml/commands/device/recording-screen/start-recording-screen.yml [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+    |
+      // TODO PHP sample
+  csharp:
+    |
+      // TODO C# sample
+
+client_docs:
+  java: "https://static.javadoc.io/io.appium/java-client/6.1.0/io/appium/java_client/screenrecording/CanRecordScreen.html#startRecordingScreen-T-"
+  javascript_wd: "https://github.com/admc/wd/blob/master/lib/commands.js#L3412"
+  ruby: )[Android]("https://www.rubydoc.info/github/appium/ruby_lib_core/Appium/Core/Android/Device#start_recording_screen-instance_method"), [iOS](""https://www.rubydoc.info/github/appium/ruby_lib_core/Appium/Core/Ios/Xcuitest/Device#start_recording_screen-instance_method"")[](
```
Comment (KazuCocoa) [inline commands-yml/commands/device/recording-screen/start-recording-screen.yml]: Here will be:
image

File: commands-yml/validator.js [hunk 5]
```diff
@@ -39,7 +39,7 @@ export default {
   'client_docs.java': {url: true},
   'client_docs.javascript_wdio': {url: true},
   'client_docs.javascript_wd': {url: true},
-  'client_docs.ruby': {url: true},
+  'client_docs.ruby': {},
```
Comment (KazuCocoa) [inline commands-yml/validator.js]: Curren spec doesn't allow input multiple URLs in the same place.
To achieve https://github.com/appium/appium/pull/11025/files#diff-3be2944a88736b0d35e8c71a33159dd4R31 , I relaxed the validation for Ruby as temporary.

**Review by jlipps — APPROVED** [review]

**Pull request merged**